app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,79 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.services.TaskService;
+import com.codesoom.assignment.utils.HttpStatus;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.*;
+import java.util.stream.Collectors;
+
+public class TaskController implements HttpHandler {
+
+    private final TaskService taskService = new TaskService();
+    private final GetController getController = new GetController(taskService);
+    private final PostController postController = new PostController(taskService);
+    private final PutContoller putController = new PutContoller(taskService);
+    private final DeleteController deleteController = new DeleteController(taskService);
```

Comment:
클래스 이름들을 가만히 읽어 봅시다. 이 클래스 이름들을 통해 우리가 알 수 있는 것이 무엇이 있나요?

Reply:
`TaskService`는 어떤 일을 할 것인지, `OOOController`는 Control한다는 것인지 명확하지가 않군요.. 관습적으로 router는 `contoller` 어미를, 로직은 `Service`라는 어미를 붙여 이름을 지어왔습니다. 

개인적으로는 `TaskService -> TaskManager`, `OOOController -> OOORouter`로 변경하면 의미가 더 명확해질 것 같은데 종립님이 보시기엔 어떤가요?!

```suggestion
    private final TaskManager taskManager = new TaskManager();
    private final GetRouter getRouter = new GetRouter(taskManager);
    private final PostRouter postRouter = new PostRouter(taskManager);
    private final PutRouter putRouter = new PutRouter(taskManager);
    private final DeleteRouter deleteRouter = new DeleteRouter(taskManager);
```

Reply:
Router를 쓰는 것도 괜찮게 느껴지는군요. 다른 언어에서는 이 단어를 쓰는 경우가 있죠. 다만 Controller라는 단어 자체가 갖는 강력한 명성이 있어서 Route 단어를 쓰면 동료들끼리의 의사소통은 Controller를 쓸 때보다 조금 더 귀찮아질 수도 있겠습니다.

TaskService를 TaskManager로 바꾸는 것은 큰 의미가 없는 활동 같아요. Manager가 Service보다 더 이해하기 쉬운 의미를 갖고 있는 것 같지가 않아서에요. 이번주는 일단 이 이름에 대한 고민은 접어두죠. 2주차나 3주차에 한번 더 생각해 볼 기회가 올 거에요.